test/functional/apps/context/_discover_navigation.js

```diff
@@ -94,5 +103,25 @@ export default function ({ getService, getPageObjects }) {
       await PageObjects.discover.waitForDiscoverAppOnScreen();
       await PageObjects.discover.waitForDocTableLoadingComplete();
     });
+
+    it('navigates to context view from embeddable', async () => {
```

Comment:
thanx for adding a functional test!

Reply:
Yes, so it also tests the `doc view` of Discover which seems to be a blind spot in functional testing of Discover (Might be because it was a separate plugin without coverage, migrated to the Discover folder). So I think the title should be `navigate to doc view from embeddable`